Assets/Scripts/Screeps3D/RoomObjects/Views/CreepView.cs

```diff
@@ -40,6 +54,8 @@ internal override void Load(RoomObject roomObject)
                 _badge.materials[0].SetTexture("EmissionTexture", _creep?.Owner?.Badge);
                 _badge.materials[0].SetFloat("EmissionStrength", .1f);
             }
+            
+            setDorito(_creep.Owner.Username == "Invader" || _creep.Owner.Username == "Source Keeper");
```

Comment:
We have the Constants file with the id of invaders and source keepers, we should probably use that :)